homeassistant/components/environment_canada/sensor.py

```diff
@@ -218,6 +226,9 @@ async def async_setup_entry(
     async_add_entities(ECSensor(coordinator, desc) for desc in SENSOR_TYPES)
     async_add_entities(ECAlertSensor(coordinator, desc) for desc in ALERT_TYPES)
 
+    aqhi_coordinator = hass.data[DOMAIN][config_entry.entry_id]["aqhi_coordinator"]
+    async_add_entities([ECSensor(aqhi_coordinator, AQHI_SENSOR)])
```

Comment:
Create an array of sensors and call `async_add_entities` once

Reply:
Fixed.